src/plugins/lua/mime_types.lua

```diff
@@ -52,29 +52,34 @@ local settings = {
   },
 
   bad_extensions = {
-    bat = 2,
-    chm = 4,
-    com = 2,
     cue = 2,
     exe = 1,
-    hta = 2,
     iso = 4,
     jar = 2,
-    lnk = 4,
-    scr = 4,
     -- In contrast to HTML MIME parts, dedicated HTML attachments are considered harmful
     htm = 1,
     html = 1,
     shtm = 1,
     shtml = 1,
     -- Have you ever seen that in legit email?
     ace = 4,
+    app = 1,
```

Comment:
It's already listed.

~~~
+ luacheck -q --no-color .
Checking src/plugins/lua/mime_types.lua           1 warning

    src/plugins/lua/mime_types.lua:66:5: (W314) value assigned to field 'app' is overwritten on line 110 before use
~~~

Reply:
Grr, my bad. Fixed, thank you!